muxpi/parse_image.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+set -e
+ARCHVARIANT="$1"
+DUT_IP="$2"
```

Comment:
Just so that it can be more consistent, make the DUT_IP and ARCHVARIANT the same number of the arguments as flash.sh